src/test-helpers/typeFactory.ts

```diff
@@ -81,6 +81,34 @@ export class TypeFactory {
 		return key.setMeta(storageEntry.creator.meta);
 	}
 
+	/**
+	 * Similar to the storageKey method above, this is specific to storageKeys
+	 * that take in double map for keys.
+	 *
+	 * @param index The id to assign the key to.
+	 * @param indexTypeOne The First InterfaceType that will be used to create the index into its new appropriate index type
```

Comment:
Does index === map key?

Reply:
`index` is the value of `indexTypeOne` changed the name to `value`